frontend/website/docs/developers/archive-node.md

```diff
@@ -2,16 +2,30 @@
 
 ## Background
 
-Since Coda nodes are by default succinct, if you need to preserve historical state, you'll want to run an `archive node`. Start the daemon using this command to run an archive process reachable at port `3086`:
-```
-coda daemon -propose-key ~/keys/my_wallet -rest 3085 -archive-port 3086
+Since Coda nodes are by default succinct, if you need to preserve historical state, you'll want to run an `archive node`. The easiest way to run the `archive node` is by the prod archive node docker-compose file.
+
+Here are required steps to achieve this:
+
+1. Clone the coda repository:
+    `git clone https://github.com/CodaProtocol/coda.git`
+2. Install `docker-compose`. You can find installation instructions for `docker-compose` on [the docker website] (https://docs.docker.com/compose/install/).
+3. Ensure that ports 3085, 3086, 9000 and 5432 are open. These ports correspond to ports for the daemon, archive, Hasura and Postgres servers to run.
```

Comment:
```suggestion
3. Ensure that ports 3085, 3086, 9000, and 5432 are open. These ports correspond to ports where the Coda daemon, archive process, Hasura, and Postgres servers will run.
```

Reply:
@wu-s-john maybe it may be useful to show how to forward ports here? Or at least add a link to instructions